scripts/adr-doctor.sh

```
@@ -0,0 +1,235 @@
+command -v bd >/dev/null || have_bd=0
+command -v jq >/dev/null || { echo "missing prerequisite: jq" >&2; exit 2; }
+[ -d "$ADR_DIR" ] || { echo "missing $ADR_DIR" >&2; exit 2; }
+if [ "$have_bd" = "0" ]; then
+  note "bd not on PATH — bd-dependent checks (decision-header bd-show, supersession edges) will be skipped"
+fi
+
+# --- count_real_files (INV-A12) ---
+note "count_real_files (INV-A12)"
+real=$(find "$ADR_DIR" -maxdepth 1 -type f -name '*.md' \
+  | grep -E '/[a-z][a-z0-9-]+-[a-z][a-z0-9-]+\.md$' \
+  | grep -vE '/[0-9]{4}-' \
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)